packages/storefront-sdk-plugins/commerce-queries/src/index.ts

```diff
@@ -1,10 +1,32 @@
-import type { WithStorefrontQuery } from '@nacelle/storefront-sdk';
+import type {
+	WithStorefrontQuery,
+	StorefrontClient
+} from '@nacelle/storefront-sdk';
+import { SpacePropertiesDocument } from './types/storefront.js';
+import type { SpaceProperties } from './types/storefront.js';
+import type { StorefrontResponse } from 'node_modules/@nacelle/storefront-sdk/dist/types/client/index.js';
```

Comment:
Steps to fix:
1. Remove this type import.

```suggestion
```

2. In `packages/storefront-sdk/src/types/plugins.ts`, change

```ts
import type { StorefrontClient } from '../index.js';
```
to
```ts
import type { StorefrontClient } from '../client/index.js';
```
3. In `packages/storefront-sdk/src/index.ts`, import the `StorefrontResponse` type and add it to the named exports.
4. In this file, import `StorefrontResponse` alongside the other type imports from `@nacelle/storefront-sdk`.